### PR TITLE
Replace X links with Bluesky links

### PR DESCRIPTION
**Pull request by michplunkett**
## Description of Changes
Since we killed the X link and are focusing on using Bluesky, I updated the site to reflect that. We did need to update the font awesome version so that it had access to the Bluesky image.

## Screenshots (if appropriate)
Before:
Screenshot 2024-11-22 at 2 29 50 PM

After:
Screenshot 2024-11-22 at 2 45 56 PM

## Tests and Linting
- [x] This branch is up-to-date with the `develop` branch.
- [x] `pytest` passes on my local development environment.
- [x] `pre-commit` passes on my local development environment.

### CODE REVIEW

File: OpenOversight/tests/test_functional.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def test_find_officer_can_see_uii_question_for_depts_with_uiis(
     ).first()
 
     dept_selector = Select(browser.find_element_by_id("dept"))
-    uii_element = browser.find_element_by_id("uii-question")
+    uii_element = browser.find_element("id", "uii-question")
```
Comment (michplunkett) [inline OpenOversight/tests/test_functional.py:166]: Before this change, the the build was failing on the 3.12 version tests.

File: OpenOversight/tests/test_functional.py [hunk 2]
```diff
@@ -162,8 +162,8 @@ def test_find_officer_can_see_uii_question_for_depts_with_uiis(
         Department.unique_internal_identifier_label.is_not(None)
     ).first()
 
-    dept_selector = Select(browser.find_element_by_id("dept"))
-    uii_element = browser.find_element_by_id("uii-question")
+    dept_selector = Select(browser.find_element(By.ID, "dept"))
+    uii_element = browser.find_element(By.ID, "uii-question")
```
Comment (michplunkett) [inline OpenOversight/tests/test_functional.py]: Before this change, the the build was failing on the 3.12 version tests.

File: OpenOversight/tests/test_functional.py [hunk 3]
```diff
@@ -135,7 +135,7 @@ def test_officer_browse_pagination(mockdata, browser, server_port):
         f"http://localhost:{server_port}/departments/{AC_DEPT}?page={last_page_index}&gender=Not+Sure"
     )
     wait_for_element(browser, By.TAG_NAME, "body")
-    page_text = browser.find_element_by_tag_name("body").text
+    page_text = browser.find_element(By.TAG_NAME, "body").text
```
Comment (michplunkett) [inline OpenOversight/tests/test_functional.py:138]: Standardized the `find_element` syntax.

**Comment by michplunkett** [comment]
There's some issue with the v3.12 tests. I'll get this addressed within the next 48 hours or so.

**Review by b-meson — APPROVED** [review]

**Pull request merged**